src/jit/compiler.h

```diff
@@ -5202,6 +5202,11 @@ class Compiler
 
     static const int MIN_CSE_COST = 2;
 
+    // Keeps tracked cse indices
+    BitVecTraits* cseTraits;
+    EXPSET_TP     cseFull;
+    EXPSET_TP     cseEmpty;
```

Comment:
Instead of "Keeps tracked cse indices"can we have three comments one for each of these new variables.